extras/create_rcsinfo.sh

```diff
@@ -6,7 +6,7 @@
 
 # default value, if git is not installed or if the source directory is
 # not under version control
-version="v2.10.0"
+version="v3.1.0-SNAPSHOT"
```

Comment:
http://xit0.org/2013/04/cmake-use-git-branch-and-commit-details-in-project/

That could simplify (eliminate) all this code, if we really want to insert the hashes in sli; on the other hand, inserting it in sli instead of c defeats the goal of making a complete and clear c api.